source/portable/NetworkInterface/STM32/NetworkInterface.c

```diff
@@ -93,6 +93,14 @@
     #define ipconfigUSE_RMII 1
 #endif
 
+#ifndef ETH_RX_BUF_SIZE
+    #define ETH_RX_BUF_SIZE 1524
+#endif
+
+#ifndef ETH_TX_BUF_SIZE
+    #define ETH_TX_BUF_SIZE 1524
+#endif
```

Comment:
For some reason CubeMX does not add these defines to `stm32h5xx_hal_conf.h` when generating code for H5 (It added them when generating for F7). The same goes for `ETH_RX_DESC_CNT` and `ETH_TX_DESC_CNT` (although some default values are set in `stm32h5xx_hal_eth.h`). 

Maybe instead of relying on these defines, we should add some NetworkInterface specific configuration options. For example:
```
niRX_DESC_CNT
niTX_DESC_CNT
niRX_BUF_SIZE
niTX_BUF_SIZE
```
These could be set in `FreeRTOSIPConfig.h`

Reply:
The updated driver uses `ipconfigNETWORK_MTU` to configure network buffer sizes so this is no longer an issue.